tutorials/demo.rst

```diff
@@ -260,19 +262,9 @@ Get help
 --------
 
 If you need help or want to request a free 30-day trial of the
-full feature set, you can write to the
+full feature set, you can write to us via the
 `support channels <https://openwisp.org/support.html>`__ or just click
-on "Contact support" as indicated in the screenshot below.
+on the tab *Contact support* as indicated in the screenshot below.
 
 .. image:: ../images/demo/contact-support.png
     :target: ../_images/contact-support.png
-
-Next steps
-----------
-
-- :doc:`Open and/or WPA protected WiFi Access Point SSID
-  <./wifi-access-point>`
-- :doc:`WiFi Hotspot, Captive Portal (Public WiFi), Social Login
-  <./hotspot>`
-- :doc:`How to Set Up a Wireless Mesh Network
-  <./mesh>`
```

Comment:
I don't think this removal was intentional. Let's not remove this.
I want to add the WPA Enterprise tutorial here too.